### PR TITLE
Releasing appium session when exiting pry with exit or ctrl-d

### LINKED ISSUE #53: arc doesn't release appium session when exiting
Don't know if it's the same for everyone but arc doesn't release the appium session once you leave the console. It means you have to kill appium and restart it to restart arc or execute test scripts.

### PR DESCRIPTION
**Pull request by mlainez**
Fixes #53 with Pry hooks.

### CODE REVIEW

File: lib/appium_console.rb [hunk 1]
```diff
@@ -50,6 +50,11 @@ def start
 
           $stdout.puts "pry #{cmd.join(' ')}"
         end
+
+        Pry.hooks.add_hook(:after_session, "Release session hook") do |output, binding, pry|
+          output.puts "Releasing appium, please wait..."
```
Comment (bootstraponline) [inline lib/appium_console.rb]: I think this is a bit more in line with the [JSONWireProtocol language](https://w3c.github.io/webdriver/webdriver-spec.html)

- `Closing appium session...`

**Comment by mlainez** [comment]
Updated. I'm still not fully picturing all the parts and conventions at play here. Is there any contributors guide somewhere that I missed and that could help me?

**Comment by bootstraponline** [comment]
Looks good, please squash to one commit then it's ready for merging.

**Comment by bootstraponline** [comment]
> Is there any contributors guide somewhere that I missed and that could help me?

Are you on the appium slack? That's where the dev team discusses appium, if not I can invite you. Better docs are [on the roadmap](https://github.com/appium/appium/blob/master/ROADMAP.md). We do have [contributor information though.](https://speakerdeck.com/jlipps/how-to-become-an-appium-contributor)

**Comment by mlainez** [comment]
Done. Thanks, got some reading to do over the week-end. I'm using appium at a mission for a customers. I'd be happy to join the slack channel to share what I'm doing.

**Comment by bootstraponline** [comment]
Invite sent.

**Comment by bootstraponline** [comment]
@mlainez [2.0.1 released to rubygems](https://rubygems.org/gems/appium_console/versions/2.0.1)

**Pull request merged**